deployments/scroll.json

```diff
@@ -0,0 +1,24 @@
+{
+  "LiFiDiamond": "0x1231DEB6f5749EF6cE6943a275A1D3E7486F4EaE",
+  "DiamondCutFacet": "0xb09e20930242327f9aC4DA95dd0c421fbE15D4db",
+  "DiamondLoupeFacet": "0x519483eEC772BA34098af187Aee7b56C77927B1a",
+  "OwnershipFacet": "0x9f8AC9248395b93220C6ABbA41d0989Aa08257Ab",
+  "WithdrawFacet": "0xf600Caa1468FFaa0B8Eef877A84152e2bDdDe2Fb",
+  "DexManagerFacet": "0xbd0B19349A4D0DF5Abb67D31659d4C5C17b78901",
+  "AccessManagerFacet": "0xdbffD7A135b332748Df2784D214c5f48eC706C3a",
+  "PeripheryRegistryFacet": "0x36eacDF8e9CF209d84B427316839800a81a2c45A",
+  "GenericSwapFacet": "0x3912De64539B960166940F2BCD55723f56A7358D",
+  "StandardizedCallFacet": "0x9D4B9784fC46496926ee4784e490F14d106D83a8",
+  "CalldataVerificationFacet": "0x9157b6550D1C3209C20aCAd12A2dC57Df34Ca48b",
+  "SymbiosisFacet": "0xb8C5408ED5f40ca8df929A50DD36b252B6b39f76",
+  "CBridgeFacet": "0xc31F4392EE89d81E7303e2c1CDE210414F39E2ea",
+  "CBridgeFacetPacked": "0x4f62E7a939C62316F76E2afC644A9A907Dc30D2d",
```

Comment:
@ezynda3 please also make sure the approval scripts for Packed Facets have been executed. I think it is missing here:
https://scrollscan.com/address/0x4f62E7a939C62316F76E2afC644A9A907Dc30D2d

Reply:
done